providers/efa/efa_trace.h

```diff
@@ -3,7 +3,11 @@
  * Copyright 2023-2025 Amazon.com, Inc. or its affiliates. All rights reserved.
  */
 
-#if defined(LTTNG_ENABLED)
+#if defined(LTTNG_ENABLED) && defined(USDT_ENABLED)
+
+#error "Only one of LTTNG_ENABLED and USDT_ENABLED allowed"
```

Comment:
This check needs to be in global CmakeList.txt and not in private header.

Thanks

Reply:
Maybe both?  I'm out of time to work on this further currently as I'll be traveling over the next 6 weeks - if someone else could progress it in the meantime, that'd be awesome.  Otherwise I'll take a look again in the new year.